client/landing/stepper/declarative-flow/internals/steps-repository/components/accent-color-control/index.tsx

```diff
@@ -79,6 +81,20 @@ const AccentColorControl = ( { accentColor, setAccentColor }: AccentColorControl
 					{ hasTranslation( 'Favorite color' ) || locale === 'en'
 						? __( 'Favorite color' )
 						: __( 'Accent color' ) }
+					{ isEnabled( 'limit-global-styles' ) && (
```

Comment:
Nitpick: What if we move the upgrade nudge markup in a separate component? This way, the main focus in this component would still be the accent-color-control logic. 

Imho, it might reduce the upgrade nudge noise from the code.